### PR TITLE
Remove unused crates from dependencies

### PR DESCRIPTION
**Pull request by tottoto**
Removes unused crates from dependencies.

### CODE REVIEW

**Comment by fulmicoton** [comment]
@Pseitz can you review?

File: stacker/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,6 @@ license = "MIT"
 [dependencies]
 murmurhash32 = "0.3"
 common = { version = "0.5", path = "../common/", package = "tantivy-common" }
-criterion = "0.4.0"
```
Comment (PSeitz) [inline stacker/Cargo.toml:10]: this should be a dev dependency

Comment (tottoto) [reply stacker/Cargo.toml:10]: Fixed.

**Comment by PSeitz** [comment]
Thanks for the PR!

**Review by PSeitz — APPROVED** [review]

**Pull request merged**